pettingzoo/test/example_envs/generated_agents_parallel_v0.py

```diff
@@ -30,7 +32,7 @@ def __init__(self, max_cycles=100, render_mode=None):
         self._act_spaces = {}
 
         # dummy state space, not actually used
-        self.state_space = gymnasium.spaces.MultiDiscrete((10, 10))
+        self.state_space = gymnasium.spaces.MultiDiscrete([10, 10])
```

Comment:
Was this a typo previously or something?

Reply:
It was giving me a type error, since MultiDiscrete "officially" only accepts `list`s and `ndarray`s.

Reply:
Oh I see, sounds good then